code/dictation.py

```diff
@@ -329,10 +329,7 @@ def dictation_insert(text: str, auto_cap: bool=True) -> str:
         # so that future dictation is properly formatted.
         text = text.replace("“", "\"").replace("”", "\"")
         actions.user.add_phrase_to_history(text)
-        # we insert the text all at once in case we have an implementation of
-        # insert that is more efficient for long strings, eg. paste-to-insert
-        actions.insert(text + (" " if add_space_after else ""))
-        if add_space_after: actions.edit.left()
+        actions.user.insert_between(text, " " if add_space_after else "")
```

Comment:
Thanks, this will allow us to override this for accessibility dictation a bit more cleanly.